mitosheet/mitosheet/step_performers/column_steps/delete_column.py

```diff
@@ -82,7 +82,8 @@ def describe( # type: ignore
         df_names=None,
         **params
     ) -> str:
-        formated_column_ids = (', '.join(column_ids))
+        # Remove None from the list, and turn the column_ids into a string
+        formated_column_ids = (', '.join(list(filter(None, column_ids))))
```

Comment:
Go back to the original code because we now handle this case in the api call

formated_column_ids = (', '.join(column_ids))